src/app/global/feature_flags.nim

```diff
@@ -23,7 +23,7 @@ const DEFAULT_FLAG_SIMPLE_SEND_ENABLED = true
 const DEFAULT_FLAG_MARKET_ENABLED = true
 const DEFAULT_FLAG_HOMEPAGE_ENABLED = true
 const DEFAULT_FLAG_LOCAL_BACKUP_ENABLED = true
-const DEFAULT_FLAG_PRIVACY_MODE_FEATURE_ENABLED = false
+const DEFAULT_FLAG_PRIVACY_MODE_FEATURE_ENABLED = true
```

Comment:
Is it really meant to be "ON" by default??? 🤯

Reply:
this is only feature flag, the actual user setting is off by default ;D